__tests__/endpoints.test.js

```diff
@@ -201,3 +200,45 @@ describe("POST api/articles/:article_id/comments ", () => {
       });
   });
 });
+
+describe("PATCH api/articles/:article_id ", () => {
+  test("PATCH 200: responds with the updated article", () => {
+    return request(app)
+      .patch("/api/articles/1")
+      .send({ inc_votes: 3 })
+      .expect(200)
+      .then((response) => {
+        const { article } = response.body;
+        expect(article.votes).toBe(103);
+      });
+  });
+  test("PATCH 404 responds with a 404 because id is valid but not found", () => {
+    return request(app)
+      .patch("/api/articles/9988")
+      .send({ inc_votes: 3 })
+      .expect(404)
+      .then((response) => {
+        expect(response.body.msg).toBe("Not Found");
+      });
+  });
+  test.skip("PATCH 200 responds with an empty body when passed an empty body", () => {
```

Comment:
An empty request body should instead be a bad request as opposed to a 200 - a 200 should signify something successful